lib/errors.js

```diff
@@ -31,7 +31,12 @@ module.exports = {
     },
     UnsupportedType: function(expectedType) {
         this.name = 'UNSUPPORTED_TYPE';
-        this.message = 'The request content didn\'t have the expected type [' + expectedType + ' ]';
+        /*jshint quotmark: double */
+        this.message =
```

Comment:
Excessive line break

Reply:
Hmmm, tricky, but fixed 5aaaacd

It is only splitting because of the jshint override which is necessary due to the apostrophe in the message.

Reply:
As far as I checked what we have now in this point is (disable marks appart):

```
        this.message = "The request content didn't have the expected type [" +
            expectedType + ' ]';
```

I think is a bit weird compared with:

```
        this.message = 'The request content didn't have the expected type [' + expectedType + ' ]';
```

or:

```
        this.message = "The request content didn't have the expected type [" + expectedType + " ]";
```

Not sure what is the problem here... it is a bit paradoxical that tools designed to make the code prettier (linter, prettier, etc.) at then end force to use which actually is ugly, don't you think?

Reply:
Fixed a65d724 - it was adding the line breaks because of the single/double quote override for jslint, if you add `prettier-ignore` you can just use single quote throughout. (and add in the `\'` for an apostrophe.